In [ ]:
import numpy as np
import scipy.stats as stats

# A

In [ ]:
data = np.array([1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29])
data_ci = stats.t.interval(0.99, 14, loc = data.mean(), scale = data.std()/np.sqrt(15))
data_ci

(1.095231668626123, 1.3821016647072102)

### 99% Confidence Interval is from 1.095 million to 1.382 million.
* Import required libraries.
* Create array with data.
* Provide confidence interval, degrees of freedom, mean and standard error of data to 'stats.t.interval' command.

Since the sample population is small (n<30) and the population standard deviation in unknown I used the t-distribution.

# B

In [ ]:
data_ci_b = stats.norm.interval(0.99, loc = data.mean(), scale = 0.2/np.sqrt(15))
data_ci_b

(1.1056514133957607, 1.3716819199375725)

### 99% Confidence Interval is from 1.106 million to 1.372 million
when population standard deviation is 0.2 million characters.